lib/routerRegistryRadix.js

```diff
@@ -133,8 +133,21 @@ RouterRegistryRadix.prototype.get = function get() {
  * @function toString
  * @returns  {String} stringified RouterRegistryRadix
  */
-RouterRegistryRadix.prototype.toString = function toString() {
+/* RouterRegistryRadix.prototype.toString = function toString() {
     return this._findMyWay.prettyPrint();
+}; */
```

Comment:
and this one too
Someone could use this serialization

Reply:
For example some cli tools which provide scaffolding. Usually with such cli you can get registered routes in terminal.

But anyway it's better to remove code not to comment it

Reply:
For display, yes we can add a function to the Anumargak. But for the time being, I've removed the comments.